### PR TITLE
epub.txt: add section about math handling

### PR DESCRIPTION
**Pull request by mb21**
contains the text from your [stackoverflow answer](http://stackoverflow.com/questions/13991893/create-an-epub-file-from-markdown-with-math) which I took the liberty to update...

### CODE REVIEW

File: epub.txt [hunk 1]
```diff
@@ -146,6 +146,12 @@ ol.toc li { list-style-type: none; margin: 0; padding: 0; }
 You can even embed fonts in the EPUB if you want; see the [User's Guide]
 under `--epub-embed-font` for instructions.
 
+## Math
+
+Pandoc has an EPUB3 writer.  It renders LaTeX math into MathML, which EPUB3 readers are supposed to support (but unfortunately still few do).  Use `pandoc -t epub3` to force EPUB3 output, as opposed to EPUB2 which is the default.
```
Comment (jgm) [inline epub.txt]: Lines in README should be wrapped to 80 columns max.

Comment (jgm) [inline epub.txt]: "unfortunately still few do" sounds a bit awkward (I know, I wrote it) -- I'd prefer "unfortunately few do"

**Comment by mb21** [comment]
right, made the changes...

**Comment by jgm** [comment]
Thanks!

**Pull request merged**